db/seeds/sandbox_data.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "faker"
+
+Faker::Config.locale = "en"
+
+logger = Logger.new($stdout)
+logger.formatter = proc do |_severity, _datetime, _progname, msg|
+  "#{msg}\n"
+end
+
+cohort = Cohort.find_or_create_by!(start_year: "2021")
+delivery_partner = DeliveryPartner.find_or_create_by!(name: Faker::Company.name)
```

Comment:
Are we ok with just a single delivery_partner shared between providers? I know we're not going to actually use them for this phase of the API, but it's probably better to model reality in structure if not in content.

Reply:
fixed